app/client/src/features/events/Questions/QuestionStats.tsx

```diff
@@ -17,12 +30,12 @@ export const QUESTION_STATS_FRAGMENT = graphql`
 
 export function QuestionStats({ fragmentRef }: QuestionStatsProps) {
     const data = useFragment(QUESTION_STATS_FRAGMENT, fragmentRef);
+    const classes = useStyles();
 
     return (
-        <CardContent>
-            <Badge showZero badgeContent={data.likedByCount}>
-                <ThumbUp color='disabled' />
-            </Badge>
+        <CardContent className={classes.stats}>
+            <ThumbUp fontSize='small' htmlColor='#8EAFFF' />
```

Comment:
Please use theme based colors.

Reply:
I tried to match the colors from the Figma, but I'll change it to the closest color on the MUI palette.

Reply:
Don't forget that you can use one of the theme colors + lighten or darken.  https://mui.com/customization/palette/#default-values for more details.